pkgs/development/python-modules/strict-rfc3339/default.nix

```
@@ -0,0 +1,18 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "strict-rfc3339";
+  version = "0.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "5cad17bedfc3af57b399db0fed32771f18fc54bbd917e85546088607ac5e1277";
+  };
+
+  meta = with stdenv.lib; {
+    homepage = "http://www.danielrichman.co.uk/libraries/strict-rfc3339.html";
```

```suggestion
    homepage = "https://github.com/danielrichman/strict-rfc3339";
```

Other hompage was 404.